spec/sidekiq/decision_review/submit_upload_spec.rb

```diff
@@ -230,5 +230,217 @@
         end.to trigger_statsd_increment('worker.decision_review.submit_upload.permanent_error')
       end
     end
+
+    context 'Supplemental Claims - when file_data exists' do
```

Comment:
These are largely the same tests as above, just with the Supplemental Claims form id instead of NOD's